myapp/models.py

```diff
@@ -21,14 +21,26 @@ class Author(models.Model):
     def __str__(self):
         return self.name
 
+
 # blog uploding model
 class Blog(BaseModel):
-    author = models.ForeignKey(Author, on_delete=models.CASCADE, related_name="blogs")
+    author = models.ForeignKey(
+        Author, on_delete=models.CASCADE, related_name="blogs")
     title = models.CharField(max_length=70)
     content = models.TextField()
 
     def __str__(self):
         return self.title
-    
+
     def get_absolute_url(self):
         return reverse('main:bloglist')
+
+
+# comments upload
+class Comments(BaseModel):
```

Comment:
Change `Comments` to `Comment`. Remember, model name must be singular not the plural.